Makefile

```diff
@@ -560,14 +560,16 @@ generate-gh-issue-templates:
 
 .PHONY: checks
 checks:
+	$(MAKE) gomoddownload
+	$(MAKE) install-tools
```

Comment:
this one cannot be cached and can add significant amount to time to each run. Not sure it should be added here